Dfc.CourseDirectory.Services.Tests/BulkUploadService/Apprenticeship/ApprenticeshipCsvRowBuilder.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Dfc.CourseDirectory.Services.Tests.BulkUploadService.Apprenticeship
+{
+    /// <summary>
+    /// Builder Pattern implementation with fluent api for building arbitrary csv row data for bulk upload testing
+    /// </summary>
+    internal class ApprenticeshipCsvRowBuilder
+    {
+        public const string DefaultTestVenueName = "A Test Venue";
+
+        private readonly Dictionary<string, string> _values = new Dictionary<string, string>(
+            ApprenticeshipCsvStructure.Fields.Select(
+                fieldName => new KeyValuePair<string, string>(fieldName, "")));
```

Comment:
```suggestion
        private readonly Dictionary<string, string> _values = ApprenticeshipCsvStructure.Fields.ToDictionary(f => f, _ => string.Empty);
```

Reply:
Nice, thanks. Done

Reply:
I kept `""` because personally I find it easier to mentally parse